src/interfaces/IMorpho.sol

```diff
@@ -112,6 +112,8 @@ interface IMorphoBase {
     /// on `transfer` and `transferFrom`. In particular, tokens with fees on transfer are not supported.
     /// - The IRM should not re-enter Morpho.
     /// - The oracle should return a price with the correct scaling.
+    /// - The oracle should not be able to go up instantly by a multiplier of more than 1/(LLTV*LIF). In particular, if
```

Comment:
```suggestion
    /// - The oracle should not be able to go down instantly by a multiplier of less than LLTV*LIF. In particular, if
```
This is the opposite because the oracle in Morpho Blue is pricing the collateral quoted in the loan token, but the problematic case is when the loan token value goes up

Reply:
you are right!